aten/src/ATen/native/cpu/Gelu.h

```diff
@@ -1,5 +1,12 @@
 #pragma once
 
+// On Windows, math.h needs to be included with _USE_MATH_DEFINES defined to
+// access constants such as M_SQRT2 and M_2_SQRTPI.
+#ifdef _WIN32
+#define _USE_MATH_DEFINES
+#include <math.h>
```

Comment:
why not include this header unconditionally?

also can we use <cmath> instead?

Reply:
I mainly wanted to limit the scope of the changes to Windows only builds.

> also can we use instead?

Sorry, I don't understand this comment 😅